config/src/lib.rs

```diff
@@ -337,6 +337,16 @@ fn xdg_config_home() -> PathBuf {
     }
 }
 
+fn xdg_config_dirs() -> Vec<PathBuf> {
+    let mut dirs = vec![xdg_config_home()];
+
+    if let Ok(d) = std::env::var("XDG_CONFIG_DIRS") {
+        dirs.extend(d.split(":").map(|s| PathBuf::from(s).join("wezterm")));
```

Comment:
Should probably use https://doc.rust-lang.org/std/env/fn.split_paths.html here rather than hard-coding the `:` splitting?

Also, consider making this XDG_CONFIG_DIRS bit guarded by `#[cfg(unix)]`

Reply:
`split_paths` makes much more sense, thanks!

I applied the guard for using XDG_CONFIG_DIRS, although it does appear to be a thing on windows as well, maybe just unset by default?